README.md

```diff
@@ -1,6 +1,9 @@
 # mitocheck_movies
 All information regarding the download and processing of Mitocheck whole movies from IDR study with accession idr0013 (screenA) ([paper](https://pubmed.ncbi.nlm.nih.gov/20360735/idr), [data](https://idr.openmicroscopy.org/webclient/?show=screen-1101), [metadata](https://github.com/IDR/idr-metadata/blob/master/idr0013-neumann-mitocheck/screenA/)). 
-Movies are downloaded with Aspera. 
+Movies are downloaded with Aspera using `download.py`. 
+Note that the full dataset is > 20 TB so the number of plates to download should be specified by changing the `plate_subset` variable.
+Then illumination correction is performed with PyBaSiC and imageJ using either `process_single_movie.py` or `process_multiple_plates.py`. 
+The number of frames to include in the shortened video can be changed with the `frames` variable and the image compression can be controlled using the `down_factor` variable. 
```

Comment:
```suggestion
You can change the number of frames to include in the shortened video using the `frames` variable and you can control the image compression using the `down_factor` variable. 
```

Switch to active voice. Also, I love these notes! They should also be in the .py file (and possible notebook) as well.